modules/server/src/main/scala/implicits.scala

```diff
@@ -39,6 +40,12 @@ trait Syntax {
   }
 }
 
+trait IOCapture {
```

Comment:
I know this is not related to the PR, but why is this called `IOCapture`? Does it make sense to rename it?